styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -0,0 +1,38 @@
+.ticker {
+  background-color: $rackspace-red;
+  width: 100%;
+}
+
+.ticker-label {
+  @include make-md-column(4);
+  color: $white;
+  font-family: $subhead;
+  padding: 45px 30px;
+  text-align: center;
+}
+
+.ticker-countdown-column {
+  @include make-md-column(4);
+}
+
+.ticker-countdown {
+  background-color: $white;
+  font-family: $subhead;
+  font-size: xx-large;
```

Comment:
We should probably stick with a standard unit for font sizes instead of generic sizes like `xx-large`